Wordle.API/Wordle.API/Models/NewScore.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Wordle.Api.Models
+{
+    public record NewScore
+    {
+        public string playerName { get; set; } = "";
+        public int attempts { get; set; }
+        public int time { get; set; }
```

Comment:
```suggestion
        public string PlayerName { get; set; } = "";
        public int Attempts { get; set; }
        public int Time { get; set; }
```
These are peropties and should be capitalized per naming conventions

Reply:
Nice catch, we'll fix that.